### DISCUSSION TITLE
获取到的视频文件是什么格式？能转换为类似使用FileReader获取的Arraybuffer吗

### DISCUSSION THREAD

**Discussion by liangshangyu** [NervJS/taro#12386] [Q & A]
Taro.chooseVideo({
        sourceType: ['album','camera'],
        maxDuration: 60,
        camera: 'back',
        success: function (res) {
             在这里需要怎么处理？才能获取到Arraybuffer。和new FileReader().result 的结果一样
        }
      })

**Comment by liuchuzhang** [comment]
用 [Taro.getFileSystemManager](https://taro-docs.jd.com/taro/docs/apis/files/getFileSystemManager)

```js
Taro.chooseVideo({
  sourceType: ['album','camera'],
  maxDuration: 60,
  camera: 'back',
  success: function (res) {
     const fs = Taro.getFileSystemManager()
      fs.readFile({
        filePath: res.tempFilePath,
        success:function(res){
           const arrayBuffer = res.data
        }
      })
  }
}
```
